unlock-app/src/stories/creator/Paywall.stories.js

```diff
@@ -4,38 +4,80 @@ import { storiesOf } from '@storybook/react'
 import Paywall from '../../pages/paywall'
 import createUnlockStore from '../../createUnlockStore'
 
-const lock = {
-  address: '0xab7c74abc0c4d48d1bdad5dcb26153fc8780f83e',
-  keyPrice: '10000000000000000000',
-  expirationDuration: '86400',
-  maxNumberOfKeys: 800,
-  outstandingKeys: 32,
+const myLock = {
```

Comment:
Why the change to `myLock`? Is that necessary for this PR?

Reply:
this was a mystery to me too, but I just realized I copied it from a unit test that uses myLock instead of lock. I can change it back :)